src/openrct2/world/footpath.h

```diff
@@ -39,34 +40,55 @@ typedef struct rct_footpath_entry {
 assert_struct_size(rct_footpath_entry, 13);
 #pragma pack(pop)
 
+// Masks for value stored in rct_tile_element.type
+enum
+{
+    FOOTPATH_ELEMENT_TYPE_FLAG_IS_QUEUE  = (0x1 << 0),
+    FOOTPATH_ELEMENT_TYPE_FLAG_IS_WIDE   = (0x1 << 1),
+    FOOTPATH_ELEMENT_TYPE_DIRECTION_MASK = (0x3 << 6),
```

Comment:
I don't find this to be an improvement over my version. I think mine is more readable.

Reply:
I still find my notation a whole lot easier to read: you know _exactly_ which bits of the values are used.

Reply:
I've changed it back to your notation now.